wordpress-mobile/WordPress-Android
# [Tooling/Fastlane] Delete unused helpers

Part of https://github.com/wordpress-mobile/release-toolkit/issues/204

This only removes the helpers that were either unused or are already present in release-toolkit.

Does not seem like much of a diff, but that change required me to compare each helper from this repo with the helpers from release-toolkit to ensure that even if they diverged over time and the diff wasn't empty, removing the local copies would not break anything.

For this first step I let the helpers and actions related to screenshots as they seem to only be present in WPAndroid. A separate step / PR will move those remaining ones into the release-toolkit and then remove them from the WPAndroid repo, but that's a larger piece of work (especially for screenshots we still need repo-specific files locally, like the `.ini` files, so the split won't be that trivial if we want it to be flexible and adapt to multiple repos in the future)

To test:

* Ensure the various actions still work (they should use the helpers from the release-toolkit now)
